pydrive2/test/test_file.py

```diff
@@ -842,6 +842,39 @@ def test_Parallel_Insert_File_Passed_HTTP(self):
         files = self._parallel_uploader(self.FILE_UPLOAD_COUNT, 10)
         self._parallel_downloader(files, 10)
 
+    # Tests for Copy file.
+    # ====================
+
+    def test_CopyFile(self):
+        drive = GoogleDrive(self.ga)
+        file1 = drive.CreateFile()
+        filename = self.getTempFile("copytestfile")
+        content = "hello world!"
+        file1["title"] = filename
+        file1.SetContentString(content)
+        pydrive_retry(file1.Upload)
+        pydrive_retry(file1.FetchMetadata)
+
+        parent_id = file1["parents"][0]["id"]
+        parent_folder = drive.CreateFile({"id": parent_id})
+
+        file1.Copy(parent_folder, new_title="copytestfile_copy")
```

Comment:
so, we copy to the same folder? can we shortcut the API in this case and make parent optional? what happens if you don't pass the parent?

also, it feels it would be more general if we create a temp dir and place the file there

Reply:
i never tried before to not pass a parent folder, but it works. Drive will place the copy in the same folder as the file.

I will also make `target_folder` optional

Reply:
also i added a new test in witch i create a temp folder and place the copy there